openedx/features/edly/tests/test_models.py

```diff
@@ -0,0 +1,94 @@
+"""
+Tests for Edly django models.
+"""
+import pytest
+from django.test import TestCase
+
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
+from openedx.features.edly.tests.factories import EdlyOrganizationFactory, EdlySubOrganizationFactory, SiteFactory
+
+
+class EdlyOrganizationTests(TestCase):
+    """
+    Tests for EdlyOrganization Model.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(EdlyOrganizationTests, cls).setUpClass()
+
+    def test_edly_organization_post_save_receiver(self):
+        """
+        Test EdlyOrganization model object creation.
+        """
+        edly_org_name = 'test org name'
+        edly_org_slug = 'testorgslug'
+        edly_organization = EdlyOrganizationFactory(name=edly_org_name, slug=edly_org_slug)
+        edly_organizations = EdlyOrganization.objects.filter(name=edly_organization.name)
+        assert len(edly_organizations) == 1
+
+    def test_edly_organization_post_update_receiver(self):
+        """
+        Test EdlyOrganization model object update.
+        """
+        edly_org_name = 'updated test org name'
```

Comment:
we can reduce these three lines to single line 
`EdlyOrganizationFactory(name= edly_org_name)`

Reply:
Done